packages/react-reconciler/src/ReactFiberErrorLogger.js

```diff
@@ -21,10 +21,6 @@ export function logCapturedError(capturedError: CapturedError): void {
   }
 
   const error = (capturedError.error: any);
-  const suppressLogging = error && error.suppressReactErrorLogging;
```

Comment:
This should check `_hasSuppressedError`. I think in order to do that we'll want to remember `ReactErrorUtils. _hasSuppressedError` at the moment we capture the error (together with the `_caughtError` itself).

Reply:
In other words we still want to *have a way* to suppress error logging.

We just don't want to use a custom property on the error object for that. Instead, we want to read a property on an object *we* created (`capturedError` is a custom object).

Reply:
I am able to get `_hasSuppressedError` property in `ReactFiberScheduler.js` and checked for the error, but in `ReactFiberErrorLogger.js` how do I get `_hasSuppressedError` property, as we have not imported `ReactErrorUtils.js`?

what I am thinking is to add a new property `hasSuppressedError` in  `capturedError` object 
```javascript
const capturedError = {
        componentName,
        componentStack,
        error,
        errorBoundary: errorBoundaryFound ? boundary.stateNode : null,
        errorBoundaryFound,
        errorBoundaryName,
        willRetry,
        hasSuppressedError,
      };
```
so that the can be found on `ReactFiberErrorLogger.js`.
is this a right approach?

Reply:
Yeah that’s what I’m thinking.